lib/class-wp-theme-json-gutenberg.php

```diff
@@ -305,35 +305,6 @@ public function __construct( $theme_json = array(), $origin = 'theme' ) {
 		}
 	}
 
-	/**
-	 * Function that migrates a given theme.json structure to the last version.
-	 *
-	 * @param array $theme_json The structure to migrate.
-	 *
-	 * @return array The structure in the last version.
-	 */
-	private static function migrate( $theme_json ) {
```

Comment:
This method has been moved to `WP_Theme_JSON_Schema::migrate` verbatim, [see](https://github.com/WordPress/gutenberg/pull/36182/files#diff-297ceb2be3349de2b92075cf28098bf3a4a5038be634b951c760c64108954f65R61).